samples/csharp_dotnetcore/02.a.echo-bot/Startup.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Configuration;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.BotBuilderSamples
+{
+    /// <summary>
+    /// The Startup class configures services and the request pipeline.
+    /// </summary>
+    public class Startup
```

Comment:
>Startup [](start = 17, length = 7)

How far are we going with Application Insights?  if the answer is "Not in this sample", then we should at least show ILogger (ie, tracing) capabilities so developers can have some diagnostics if/when the bot fails so support has something to guide the user for exceptions.
For ILogger, you could add something like:
[https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L145](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L145)
[https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L81](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L81)
[https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L86](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/csharp_dotnetcore/03.welcome-user/Startup.cs#L86)

If we want to double down on App Insights, the global exception handler could log into App Insights (the LUIS with App Insights samples would be a good one to copy).

Reply:
makes sense.  I'll add ILogger support

Reply:
added ILogger support